src/containers/EnterpriseApp/EnterpriseApp.test.jsx

```diff
@@ -118,7 +118,7 @@ describe('EnterpriseApp', () => {
         document.body.removeChild(div);
       }
     });
-    it('handles location change properly', () => {
+    it.skip('handles location change properly', () => {
```

Comment:
This test is failing here, yeah?

```
// ensure focus is set on content wrapper
expect(document.activeElement.className).toEqual('content-wrapper');
```

So long as the content wrapper is actually getting focus on route change, skipping this test is fine for now :)

Reply:
Yeah, I can't figure it out. It's working via manual testing. Ideas welcome.